app/main.py

```diff
@@ -1,25 +1,94 @@
-from abc import ABC
+from abc import ABC, abstractmethod
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(self, instance: Any, owner: Any) -> dict:
+        return instance.__dict__[self.name]
+
+    def __set__(self, instance: Any, value: int) -> None:
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError(f"{self.name} must be between"
+                             f" {self.min_amount} and {self.max_amount}")
+        instance.__dict__[self.name] = value
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        self.name = name
 
 
 class Visitor:
-   pass
+    age = IntegerRange(0, 200)
+    weight = IntegerRange(0, 1000)
+    height = IntegerRange(0, 300)
+
+    def __init__(self, name: int, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
+
+    @abstractmethod
+    def can_access(self, visitor: Visitor) -> None:
+        pass
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    def __init__(self) -> None:
+        super().__init__(age=IntegerRange(4, 14),
+                         weight=IntegerRange(20, 50),
+                         height=IntegerRange(80, 120))
+
+    def can_access(self, visitor: Visitor) -> tuple:
+        return (
+            self.age.min_amount
+            <= visitor.age
+            <= self.age.max_amount
+            and self.weight.min_amount
+            <= visitor.weight
+            <= self.weight.max_amount
+            and self.height.min_amount
+            <= visitor.height
+            <= self.height.max_amount
+        )
```

Comment:
Move the implementation inside `Slide` class. You don't need to check every attribute, you can just catch ValueError that is risen in case attribute doesn't fit into allowed range